web/html/src/components/input/InputBase.js

```diff
@@ -5,6 +5,8 @@ const { FormGroup } = require('./FormGroup');
 
 export type Props = {
   name: string,
+  // eslint-disable-next-line
```

Comment:
Why do we need this disable?

Reply:
Since that prop is defined in InputBase and used in Form eslint/flow is complaining that it is not used. If you have a smarter way to kill that error that would be great.